extensions/_beacons/http_status.py

```diff
@@ -55,13 +55,16 @@ def validate(config):
     if not sites:
         return False, ('You neglected to define any sites')
 
-    for site, settings in sites.iteritems():
+    for site, settings in sites.items():
         if required_site_attributes.isdisjoint(set(settings.keys())):
             return False, ('Sites for {} beacon requires {}'.format(__virtualname__,
                                                                     required_site_attributes))
-            if optional_site_attributes and optional_site_attributes.isdisjoint(set(settings.keys())):
+        try:
+            if optional_site_attributes.isdisjoint(set(settings.keys())):
                 return False, ('Sites for {} beacon requires {}'.format(__virtualname__,
                                                                         optional_site_attributes))
+        except NameError:
+            log.info('No optional attributes defined')
```

Comment:
I don't follow what this is intended to catch. This is still enforcing that the optional attributes end up being required...